app/code/Magento/ImportExport/Block/Adminhtml/Export/Filter.php

```diff
@@ -418,14 +437,45 @@ public function getRowUrl($row)
         return false;
     }
 
+    /**
+     * @return array
+     * @throws \Magento\Framework\Exception\LocalizedException
+     */
+    public function getDisabledAttrs()
+    {
+        $disabledAttrs = [];
+        $productTypes = $this->exportConfig->getEntityTypes(CatalogProduct::ENTITY);
+        foreach ($productTypes as $productTypeConfig) {
+            if (!($model = $this->typeFactory->create($productTypeConfig['model']))) {
+                throw new \Magento\Framework\Exception\LocalizedException(
```

Comment:
In my opinion, collecting disabled attributes is not critical operation enough to stop the process execution and throw an exception, can we simply `continue` in case we are unable to get disabled attributes for a type?

Reply:
yeah , we can continue the process and not break it in case we'r unable to get these attrs .. 
:)
updating these changes .